### PR TITLE
Added some checks according to changed iOS9 API.

### PR DESCRIPTION
**Pull request by nsndeck**

### CODE REVIEW

**Comment by enchev** [comment]
:+1:

File: apps/tests/ui/list-view/list-view-tests.ts [hunk 1]
```diff
@@ -675,7 +675,12 @@ function getTextFromNativeElementAt(listView: listViewModule.ListView, index: nu
         return (<android.widget.TextView>nativeElement).getText() + "";
     }
     else if (listView.ios) {
-        return listView.ios.visibleCells()[index].contentView.subviews[0].text + "";
+        if (utils.ios.MajorVersion <= 8) {
+            return listView.ios.visibleCells()[index].contentView.subviews[0].text + "";
+        }
+        else if (utils.ios.MajorVersion > 8) {
```
Comment (ligaz) [inline apps/tests/ui/list-view/list-view-tests.ts]: Do you need the `if` here?

**Comment by ligaz** [comment]
Looking at the code is it possible to monkey patch those in the prototype as well?

**Comment by fealebenpae** [comment]
@ligaz: Replacing an existing property on the prototype of an Objective-C class will try to swizzle the native method instead. Besides, messing with the prototype of a system object can ruin it for other consumers of that API

**Comment by nsndeck** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by dtopuzov** [comment]
:+1:

**Pull request merged**